assets/js/src/frontend/navigation.js

```diff
@@ -49,17 +49,16 @@ export const repositionDropdowns = () => {
 			rightDist = bounding.left;
 
 		if (rightDist < 0) {
-			left = isRTL ? 'auto' : 0;
-			right = isRTL ? '-100%' : 'auto';
-			dropDown.style.right = right;
-			dropDown.style.left = left;
+			dropDown.style.right = isRTL ? '-100%' : 'auto';
+			dropDown.style.left = isRTL ? 'auto' : 0;
 		}
 
 		if (rightDist + bounding.width >= windowWidth) {
-			right = isRTL ? 0 : '100%';
-			left = 'auto';
-			dropDown.style.right = right;
-			dropDown.style.left = left;
+			dropDown.style.right = isRTL ? 0 : '100%';
+			dropDown.style.left = 'auto';
+		}
+		if (typeof menuCalcEvent !== 'undefined') {
```

Comment:
@preda-bogdan I'm thinking it would be better to dispatch the event only once after all the dropdowns are properly aligned. Right now it dispatches after aligning each dropdown. What do you think? 🐑

Reply:
I've moved the trigger outside of the loop. Thank you for catching this.